packages/eslint-plugin/tests/rules/no-floating-promises.test.ts

```diff
@@ -140,9 +140,8 @@ async function test() {
 }
     `,
     `
+declare const promiseValue: Promise<number>;
 async function test() {
-  declare const promiseValue: Promise<number>;
```

Comment:
These changes are just because declare is not syntactically allowed in that position. Just some hygiene, not relevant to the changes.